### PR TITLE
spanish translations for the reset password bundle

### PR DESCRIPTION
**Pull request by jrushlow**
_Translations were AI Generated by AWS Translate Machine Learning Platform and need to be Validated by humans_

Includes all of the Spanish translations for Symfony Cast's Reset Password Bundle in PR #1100

### CODE REVIEW

File: translations/messages+intl-icu.es.xlf [hunk 1]
```diff
@@ -366,6 +366,63 @@
...
+                <target>Debido a que la dirección de correo electrónico introducida no existe, mostramos esta plantilla, en lugar de revelar que el usuario no existe.</target>
+            </trans-unit>
+            <trans-unit id="reset_password.check_email.title">
+                <source>reset_password.check_email.title</source>
+                <target>Correo electrónico de restablecimiento de contraseña enviado</target>
+            </trans-unit>
+
+            <trans-unit id="reset_password.email.title">
+                <source>reset_password.email.title</source>
+                <target>¡Hola! ¡Hola!</target>
```
Comment (rperezmira) [inline translations/messages+intl-icu.es.xlf:402]: Should be just one `Hola!` it's twice

File: translations/messages+intl-icu.es.xlf [hunk 2]
```diff
@@ -366,6 +366,63 @@
...
+            </trans-unit>
+            <trans-unit id="reset_password.email.body">
+                <source>reset_password.email.body</source>
+                <target><![CDATA[Para restablecer tu contraseña, visita <a href="{email_link}">aquí</a> Este enlace caducará en {expires} hora (s).]]></target>
+            </trans-unit>
+            <trans-unit id="reset_password.email.closing">
+                <source>reset_password.email.closing</source>
+                <target>¡Salud!</target>
```
Comment (rperezmira) [inline translations/messages+intl-icu.es.xlf:410]: Should be `Saludos!` instead of `Salud`

File: translations/messages+intl-icu.es.xlf (hunk 1 above)
Comment (jrushlow) [reply translations/messages+intl-icu.es.xlf]: Thank you!

File: translations/messages+intl-icu.es.xlf (hunk 2 above)
Comment (jrushlow) [reply translations/messages+intl-icu.es.xlf]: Thank you!

**Comment by javiereguiluz** [comment]
Closing because the related #1100 proposal was closed. Sorry! And thank you!

**Pull request closed without merging**